plugins/copr.py

```diff
@@ -454,10 +461,8 @@ def _guess_chroot(self):
         return chroot
 
     def _download_repo(self, project_name, repo_filename, chroot=None):
-        if chroot is None:
-            chroot = self._guess_chroot()
```

Comment:
Looks like you removed this unintentionally

Reply:
No, that was intentional, at this point, it can't happen that the chroot is None.

Reply:
Nitpicking, but then you could also remove the `=None` from the function arguments :slightly_smiling_face:. Otherwise LGTM now!

Reply:
Good point, I removed it.